Kudu.Core/Deployment/Generator/SiteBuilderFactory.cs

```diff
@@ -159,6 +159,10 @@ private ISiteBuilder ResolveNonAspProject(string repositoryRoot, string projectP
             {
                 return new RubySiteBuilder(_environment, perDeploymentSettings, _propertyProvider, repositoryRoot, projectPath);
             }
+            else if (IsPHPSite(sourceProjectPath))
```

Comment:
I think we should only do this for Linux, or we might break existing PHP sites on Windows.